doc/HOWTO_oracle.md

```diff
@@ -4,145 +4,145 @@ Follow these instructions to set up the Oracle database to run in Docker, but th
 
-* Install into `/Applications`
-
-* Start the oracle container:
+### Start an Oracle container
 
 ```
-mkdir $HOME/oracle_data
-docker run -p 1521:1521 --name nucore_db -v $HOME/oracle_data:/u01/app/oracle sath89/oracle-12c
-# This will take several minutes as the database initializes.
-# Wait for "Database ready to use. Enjoy! ;)"
-```
+# Feel free to change this to a more suitable directory on your computer.
+# Specifying a directory for the container allows you to reuse the development
+# data, and skips the initialization of an empty database every time you
+# restart the container.
+ORACLE_DATA_DIR=$HOME/oracle_data
 
-`$HOME/oracle_data` is just a recommended location where your oracle data files will
-be saved. This will allow them to remain persistent even across re-creations of
-the container.
+# Run the Oracle image as a container named oracle, storing data
+# in the directory specified above. When you run this for the first time,
+# it will take a while to download the image and initialize the database.
+#
+# When the output in the terminal includes a line like:
+#
+#   Done ! The database is ready for use .
+#
+# you can use the database
+docker run \
+  --interactive \
+  --name oracle \
+  --publish 1521:1521 \
+  --tty \
+  --volume $ORACLE_DATA_DIR:/ORCL \
+  store/oracle/database-enterprise:12.2.0.1
+```
 
-* Next time you want to start the server, run:
+When you no longer need the database, in another terminal tab run:
 
 ```
-docker start nucore_db
-# wait, it sometimes takes a few minutes to come up
-# "ORA-01033: ORACLE initialization or shutdown in progress" means wait.
+docker stop oracle
 ```
 
-## Setting up the Oracle Client Drivers
-
-##### Install Oracle 11.2 Instant Client
-
-* Download Basic, SqlPlus, and SDK from: `http://www.oracle.com/technetwork/topics/intel-macsoft-096467.html`
-
-* Install with:
+The next time you need the database, start it just by running:
 
 ```
-sudo mkdir -p /opt/oracle/
-cd /opt/oracle
-
-sudo mv ~/Downloads/instantclient-* /opt/oracle
-sudo unzip instantclient-basic-macos.x64-12.1.0.2.0.zip
-sudo unzip instantclient-sdk-macos.x64-12.1.0.2.0.zip
-sudo unzip instantclient-sqlplus-macos.x64-12.1.0.2.0.zip
+docker start --interactive oracle
+```
 
-sudo ln -s instantclient_12_1 instantclient
+## Setting up the Oracle Client Drivers
 
-cd instantclient
-sudo ln -s libclntsh.dylib.12.1 libclntsh.dylib
-sudo ln -s libocci.dylib.12.1   libocci.dylib
+### Install Oracle Instant Client
 
-sudo ln -s /opt/oracle/instantclient/sqlplus /usr/local/bin/sqlplus
-```
+1. Enable the homebrew tap for Oracle Instant Client by running `brew tap InstantClientTap/instantclient`.
+1. Run each of these commands, following the displayed instructions for how to download the appropriate .zip files from Oracle’s website. After you download the files and place them in Homebrew’s cache directory, brew will take care of the rest:
+    ```
+    brew install instantclient-basic
```

Comment:
I didn't know you could use `brew` for this. This makes things much easier. 🎉

Reply:
Yeah, you still have to download the files manually and put them in the right directory (which will be displayed in the instructions), but it’s been pretty nice!

Reply:
Makes sense. I was wondering how they were getting around the distribution restrictions.